content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+---
+title: The Istio network configuration objects and your application
+description: Describes Istio's network configuration objects and their functionality.
+weight: 30
+keywords: [network, virtual service, gateway, ingress, egress, service entry]
+aliases:
+---
+
+Cloud applications, unlike monolithic applications, are comprised of a
+collection of connected services. Throughout this document a service should be
+understood as an addressable destination workload inside the mesh. These
+services can connect to other services or to external resources. The Istio
+configuration objects allow you to configure traffic routes to and from
+services to secure the traffic between them. Other configuration objects
+provide telemetry and monitoring configuration options but this document
+focuses on the Istio's network configuration objects.
+
+The Istio network API defines four configuration objects: gateways, virtual
+services, destination rules, and service entries.
+The diagrams on this page don't show data plane traffic. Unless otherwise
+indicated, the arrows in the diagrams portray the traffic routes that the
+network configuration objects configured using destination rules. The diagrams
+show the possible control plane connections between the Envoy proxies of the
+network configuration objects.
+
+The following diagram shows the **configurable traffic routes** that the
+network configuration objects allow for a basic mesh with two different
+versions of a service:
+
+{{< image width="80%"
+    link="./net-config-1.svg"
+    caption="Istio network configuration objects overview"
+    >}}
+
+This document describes the key configurations each configuration object
+enables. Additionally, this document provides high-level examples of how the
+Istio network API can help you handle internal, incoming, and outgoing traffic.
+Lastly, the document discusses the benefits of using the Istio network API for
+your cloud applications.
+
+# The Istio network configuration objects
```

Comment:
For our docs, the highest-level header is H2. So this should be ##

Reply:
Done